NSS-Day-Cohort-68/rare-api-rare-team-2-1
# Feature/create category/api TICKET #42

Added backend logic for POST-ing new categories to the database via postman until front end logic is completed. 

To Test: 
1. Paste the following URL into postman and switch the method to POST. 
 http://localhost:8088/categories

2. Paste the following block into the raw body.
{
    "label": "The Roman Empire and their discovery of Python"
}

3. Check your database path and ensure that the POST was successful. Thanks. 

TICKET #42
